fiftyone/core/annotation/validate_label_schema.py

```
@@ -0,0 +1,590 @@
+def _validate_default(field_name, value, _type, _range=None, values=None):
+    exception = ValueError(
+        f"invalid 'default' setting '{value}' for field '{field_name}'"
+    )
+    if isinstance(value, _type):
+        if _range is not None and (value < _range[0] or value > _range[1]):
+            raise exception
+
+        if values is not None and (value not in values):
+            raise exception
+
+        if _type == dict:
```

Ruff (0.14.7)

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)